packages/remix-server-runtime/__tests__/handler-test.ts

```diff
@@ -16,9 +16,7 @@ describe("createRequestHandler", () => {
       assets: {} as any,
       entry: { module: {} as any },
       // @ts-expect-error
-      future: {
-        v2_meta: false,
-      },
+      future: {},
```

Comment:
```suggestion
```

Reply:
This needs to be here to satisfy the type check for `createRequestHandler` since it's expecting a resolved `future` object.